scene/gui/tree.cpp

```diff
@@ -4202,6 +4202,12 @@ void Tree::item_deselected(int p_column, TreeItem *p_item) {
 
 	if (select_mode == SELECT_MULTI || select_mode == SELECT_SINGLE) {
 		p_item->cells.write[p_column].selected = false;
+	} else {
+		if (select_mode == SELECT_ROW) {
```

Comment:
This can be combined into one block
```suggestion
	} else if (select_mode == SELECT_ROW) {
```